src/xenia/kernel/xam/user_profile.cc

```diff
@@ -18,8 +18,8 @@ namespace kernel {
 namespace xam {
 
 UserProfile::UserProfile() {
-  xuid_ = 0xBABEBABEBABEBABE;
-  name_ = "User";
+  xuid_ = 0xE000BABEBABEBABE;
+  name_ = "Xenia";
```

Comment:
This should probably be moved to a separate issue/PR as it isn't related to this PR

Reply:
The Xuid change is most definitely related. The name, admittedly, is just cosmetic and could be left out from a merge. I'll probably leave it as-is until Rick addresses it.